yb-voyager/src/srcdb/ora2pg_export_data.go

```diff
@@ -89,8 +89,8 @@ func ora2pgExportDataOffline(ctx context.Context, source *Source, exportDir stri
 	exportDataStart <- true
 
 	err = exportDataCommand.Wait()
-	log.Infof("ora2pg STDOUT: %s", outbuf.String())
-	log.Errorf("ora2pg STDERR: %s", errbuf.String())
+	log.Infof("ora2pg STDOUT: %q", outbuf.String())
```

Comment:
This will make the log messages unreadable e.g. all new lines will be replaced by `\n`. Any particular reason for changing the format specifier here?